docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1190,6 +1190,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/core/defaultPrettyPrint`|boolean|A boolean flag which indicates whether "Pretty Printing" should be used by default. Applies to both XML and JSON.|
 |`fhirServer/core/tenantIdHeaderName`|string|The name of the request header that will be used to specify the tenant-id for each incoming FHIR REST API request. For headers with semicolon-delimited parts, setting a header name like `<headerName>:<partName>` will select the value from the part of header `<headerName>`'s value with a name of `<partName>` (e.g. setting `X-Test:part1` would select `someValue` from the header `X-Test: part1=someValue;part2=someOtherValue`).|
 |`fhirServer/core/dataSourceIdHeaderName`|string|The name of the request header that will be used to specify the datastore-id for each incoming FHIR REST API request. For headers with semicolon-delimited parts, setting a header name like `<headerName>:<partName>` will select the value from the part of header `<headerName>`'s value with a name of `<partName>` (e.g. setting `X-Test:part1` would select `someValue` from the header `X-Test: part1=someValue;part2=someOtherValue`).|
+|`fhirServer/core/handling`|string|The default handling mode of the server (`strict | lenient`) and whether to honor client handling preferences passed through the Prefer header or not (`strict vs strict-only`)|
```

Comment:
maybe prefer-handling is more accurate?

Reply:
well this is the server config.  the client can pass their preference in the Prefer header, but this config setting is for controlling what the server uses.  could be `handling-mode` or some such, but i thought just `handling` was fine.